modules/vaos/app/services/ccra/referral_service.rb

```diff
@@ -30,7 +30,8 @@ def get_vaos_referral_list(icn, referral_status)
         Rails.logger.info("CCRA Referral List - Content-Type: #{response.response_headers['Content-Type']}, " \
                           "Body Class: #{response.body.class}, Body Preview: #{body_preview}...")
 
-        data = response.body.is_a?(String) ? JSON.parse(response.body) : response.body
+        # Note JSON.parse is only used for betamock responses
+        data = response.body.is_a?(String) ? JSON.parse(response.body, symbolize_names: true) : response.body
```

Comment:
Note: A real or VCR response is an object, which betamocks bypasses the Faraday middleware and comes as a string.